### ISSUE TITLE
STABS: Imported type conflicts

### ISSUE THREAD

**Issue by VelocityRa** [chaoticgd/ghidra-emotionengine-reloaded#17]
Game: https://hiddenpalace.org/Sly_Cooper_and_the_Thievius_Raccoonus_(May_19,_2002_prototype)

![image](https://user-images.githubusercontent.com/6632271/219035241-3e84f866-14ee-4157-bfe0-d1245860f9c7.png)

Also on import:
```
STABS> Type lookup failed: FE
STABS> Type lookup failed: FE
STABS> Type lookup failed: FE
STABS> Type lookup failed: FE
STABS> Type lookup failed: ASO
STABS> Type lookup failed: rtx_def
```

Possibly related to #16.

**Comment by chaoticgd** [comment]
I've mostly fixed this in ccc. A little more work is needed on the Java side then this issue should be resolved completely.

**Comment by chaoticgd** [comment]
This is fixed by #18.
